src/pages/Checkout.page.js

```diff
@@ -0,0 +1,52 @@
+const { ShopingCartPage } = require("./ShopingCart.page");
+const { createRandomUser }  = require("../helpers/fakeData");
+const { parsePrice } = require("../helpers/parsePrice");
+
+
+export class CheckoutPage extends ShopingCartPage {
+  url = "checkout-step-one.html";
+
+  get firstName() {
+    return this.page.locator('[id="first-name"]');
+  }
+
+  get lastName() {
+    return this.page.locator('[id="last-name"]');
+  }
+
+  get postalCode() {
+    return this.page.locator('[id="postal-code"]');
+  }
+  get continueButton() {
+    return this.page.locator('[id="continue"]');
+  }
+
+  get tax() {
+    return this.page.locator('.summary_tax_label')
+  }
+
+  get totalPrice() {
+    return this.page.locator('.summary_total_label')
+  }
+
+  async fillData() {
```

Comment:
The user data should be a parameter for this method:
```suggestion
  async fillData(user) {
```
This will allow to fill any user info on the checkout page